### PR TITLE
feat: basic structure page

### PR DESCRIPTION
**Pull request by Uchita-Hikaru**

### CODE REVIEW

File: src/components/Main.astro [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 <main
   transition:animate="fade"
-  class="container mx-auto px-24 py-8 max-sm:px-5"
+  class="px:6 container mx-auto py-8 max-sm:px-5 lg:px-24"
```
Comment (avevotsira) [inline src/components/Main.astro:7]: leave this to default , we'll fix everything later

**Comment by avevotsira** [comment]
Let me take this over

**Comment by avevotsira** [comment]
Will check this tomorrow

**Comment by avevotsira** [comment]
request changed will make this too much to refactor , we should start from scratch. will reuse code later

**Comment by Uchita-Hikaru** [comment]
okay

**Pull request closed without merging**